IUBLibTech/ngao
# Ar 22 limit repositories to user

# Summary 
Add user table to manage users - delete, update role, add repository to manage
Part of ticket AR-22 

# Related Issue
https://bugs.dlib.indiana.edu/browse/AR-22

# Expected Behavior

In order to restrict which repositories a user can see, we first need to set up a place for admin users to assign repositories to the user that they can manage 

# Screenshots / Video
Include screenshots or video demonstrating how this feature works
![image](https://user-images.githubusercontent.com/18175797/80326457-b3b1e300-87ed-11ea-9a76-81b2ca26cd9f.png)


# Testing / Reproduction instructions

1. Login as admin - you should be directed to the admin page
2. Select the Manage Users tab
3. You can delete users, manage user repositories and make admin/manager
DO NOT DELETE ANY OF THE CURRENT USERS! ONLY DELETE A USER THAT YOU HAVE CREATED.

# Deployment

Will deploy to staging when merged
